### ISSUE TITLE
Mobile - update mobile equipment + apps (10/15 flex)

### ISSUE THREAD

**Issue by ka78** [tnc-ca-geo/gd_intern#77]
Context: we use iPads for mobile data collection. Every year the iPads, apps and GPS receivers should be updated to ensure they're running the most current version and firmware.  Katie will show you where the equipment is and share our Apple ID creds, in person.
1.  update iOS on iPads
2. ensure apps are updated (Collector, Explorer, Avenza, Dual, etc)
3. bluetooth pair with every GPS receiver to download firmware updates (need to be in a location that offers both wi-fi and enough satellite visibility to get a signal) - please pair with all new Dual receivers AND the new iSxblue receiver (hasn't yet been out of the box).

**Comment by connorshank** [comment]
Update on progress:

iPad minis 4 and 5, as well as TNC Chico Tablet-3 have all been fully updated and can link with all six of the Dual GPS receivers (3 XGPS150s and 3 XGPS160s).  Although the Tablet is charging very slowly, and could use a new charging cord.

The third iPad mini (clear case, unlabeled) is having updating errors and is charging very slowly, it also appears to have trouble linking with some of the GPS receivers via bluetooth.   This could mean the iPad itself is faulty and it may need to be taken to an Apple store in order to be fully repaired and updated.

**Comment by ka78** [comment]
Thanks for the update @connorshank,
-- is the charging cord bad? or do we just need an additional one?
-- there is another iPad mini on Santa Cruz island, it'll come back to use in mid-September
-- I recall there was one iPad that had a "bad" sticker on it, is this the one you're referring to that's having update errors?
-- are iPad minis 1 + 2 accounted for (actually one of them is on the island)?

**Comment by connorshank** [comment]
There was no cord for the large iPad, luckily Carolyn had a spare one at the front desk that I've been using, however it's a little worse for wear and frayed at the ends.   As for iPad minis 1 & 2, if one is on Santa Cruz then I assume the bad iPad is the 4th one, however it had no label so I don't know which it is specifically (1 or 2).

**Comment by ka78** [comment]
re-opening this issue >> I'd like to make sure all of our Dual GPS units have the latest firmware.  I saw this in a recent TNC Collector group forum: 
"Dual XGPS device – they are currently running two different firmware versions, a 1.3.4 and a 2.1.0 – the 1.3.4 works fine but requires a “helper app” (like Bluetooth GPS on your app store of choice) to communicate with Collector.  On Android devices this requires enabling mock locations through developer mode, which is a bit of a pain.  Upgrading to the 2.1.0 will allow you to choose your GPS device from within Collector itself and bypass the mock locations process (although it still kicks out an “estimated accuracy will be used” error which you can safely ignore)."

Thx, Katie

**Comment by connorshank** [comment]
Katie,   Just an update on the GPS units:   The two XGPS160 units we have in the office (the third is the one we mailed to Rodd) are both on a firmware version of 3.2.3 or greater.   The Status app on the iPads has been updated, but now does not appear to be able to directly connect with XGPS150 units, although the iPad appears to be able to connect with the 150 units when you go into bluetooth settings.   I'm not sure why this is, there doesn't seem to be any information on this in the GPS user's manuals.    In addition, when you try to open the Dual Status app on the iPad, it gives an error saying " 'Status Tool' Needs to be Updated - The developer of this app needs to update it to work with iOS 11" however, it is possible to open the app via the app store.    I'm encountering a lot of weird problems here, and I will try to test the GPS in the field (i.e. the porch) to see if they are functioning properly.
